status-im/status-react
# Add link preview for twitter links

fixes #11425

### Summary

Allows unfurling for Twitter links - Twitter previews can now be displayed in the chat view

Most of the changes are done in status-go:
https://github.com/status-im/status-go/pull/2298

This PR only makes it so that when thumbnailUrl is empty, link preview doesn't display an image view

<details>
  <summary>Expand to see screenshot. ⬇</summary>

![twitter-unfurling](https://user-images.githubusercontent.com/10310633/128092500-488e7df2-0627-4a93-a970-bb6ba3d4d940.jpg)

</details>


### Review notes

check out https://github.com/status-im/status-go/pull/2298

### Testing notes

make sure you're running status-go with these changes:
https://github.com/status-im/status-go/pull/2298

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- 1-1 chats
- public chats
- group chats

##### Non-functional

- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test

- Open Status
- Open a chat
- Send a twitter link
- Receive a twitter link
- Tap on link preview and make sure it takes you to the right tweet/profile/etc.
- Scroll the list and see if scrolling performance is affected


status: ready
